games-misc/an-anime-game-launcher-bin/an-anime-game-launcher-bin-2.3.8-r2.ebuild

```diff
@@ -39,25 +38,48 @@ src_unpack() {
 src_prepare(){
 	chmod +x ${P}.AppImage
 	./${P}.AppImage --appimage-extract || die "Extraction Failed"
-	chrpath -d "squashfs-root/public/discord-rpc/discord-rpc" || die "Patching Library Failed"
 	default
+
 	mv "squashfs-root/public/icons/256x256.png" "${PN}.png" || die
 	mv "squashfs-root/AppRun" "${PN}" || die
 	mv "squashfs-root/an-anime-game-launcher.desktop" "${PN}.desktop" || die
 }
 
 src_install(){
+
 	insinto "/usr/lib/${PN}"
 	doins "squashfs-root/resources.neu"
 	exeinto "/usr/lib/${PN}"
 	doexe "squashfs-root/an-anime-game-launcher"
-	doins -r "squashfs-root/public"
+
+	insinto "/usr/lib/${PN}/public" # specify path for public folders
+
+	#copy all files/folders from squashfs-root/public
+	doins -r "squashfs-root/public/discord-rpc"
```

Comment:
Why do we need to copy those files over one by one? the `-r` flag does a directory recursively, the line before copied everything from the `public` folder into `/usr/lib/${PN}`. Do we need to do it one by one?

Reply:
This was for safeties sake while i was writing the ebuild, it can be rolled back to how it was